docs/config/README.md

```diff
@@ -2082,6 +2083,45 @@ The module will be shown if any of the following conditions are met:
 format = "via [🏎  $version](red bold)"
 ```
 
+## Status
+
+The `status` module displays the exit code of the previous command.
+The module will be shown only if the exit code is not `0`.
+
+### Options
+
+| Variable   | Default                       | Description                                            |
+| ---------- | ----------------------- ------| ------------------------------------------------------ |
```

Comment:
The formatting for "Options" is messed up, probably due to the extra space.

<img width="865" alt="format" src="https://user-images.githubusercontent.com/5209296/93022321-ccf9d700-f5b6-11ea-8ede-11e04cc259d6.png">

<img width="710" alt="status" src="https://user-images.githubusercontent.com/5209296/93022310-ba7f9d80-f5b6-11ea-8f56-268035bf757d.png">

Reply:
Fixed the out-of-space-space and missing space in the end, but the spacing in the symbol line seems to come from the chosen character and how big it is displayed in a specific font:

diff on the commandline:
![image](https://user-images.githubusercontent.com/890156/93026397-fdb52d00-f605-11ea-8e75-fe7228a332e8.png)

vs 

jetbrains
![image](https://user-images.githubusercontent.com/890156/93026412-1ae9fb80-f606-11ea-8b53-3e7843ea5121.png)

vs 
gitkraken

![image](https://user-images.githubusercontent.com/890156/93026422-263d2700-f606-11ea-90be-50ffe72c4946.png)